test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +69,12 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertAreNULL(Object... objects) {
```

Comment:
Coding standard violation: Abbreviations and acronyms should not be uppercase when used as a part of a name. Hence `NULL` should be `Null`